esp/public/media/scripts/onsite/teacher_checkin.js

```diff
@@ -135,6 +135,12 @@ $j(function(){
         $msg.text('Texting teacher...');
     });
 
+    $j(".text-all").click(function(){
+        $j(".text").click()
```

Comment:
You may want to put a confirm here, since this could be texting dozens of people.